godal.cpp

```diff
@@ -806,6 +806,51 @@ void godalRasterHistogram(cctx *ctx, GDALRasterBandH bnd, double *min, double *m
 	godalUnwrap();
 }
 
+void godalComputeRasterStatistics(cctx *ctx, GDALRasterBandH bnd, int bApproxOK, double *pdfMin, double *pdfMax, double *pdfMean, double *pdfStdDev){
+  godalWrap(ctx);
+  CPLErr ret = CE_None;
+  ret = GDALComputeRasterStatistics(bnd, bApproxOK, pdfMin, pdfMax, pdfMean, pdfStdDev, nullptr, nullptr);
+  if (ret != 0) {
+    forceCPLError(ctx,ret);
+  }
+  godalUnwrap();
+}
+
+void godalGetRasterStatistics(cctx *ctx, GDALRasterBandH bnd, int bApproxOK, double *pdfMin, double *pdfMax, double *pdfMean, double *pdfStdDev){
+  godalWrap(ctx);
+  CPLErr ret = CE_None;
+  if (bApproxOK == 1) {
+    ret = GDALGetRasterStatistics(bnd, 1, 0, pdfMin, pdfMax, pdfMean, pdfStdDev);
+  } else {
+    ret = GDALGetRasterStatistics(bnd, 0, 0, pdfMin, pdfMax, pdfMean, pdfStdDev);
+  }
+  if (ret != 0) {
+    forceCPLError(ctx,ret);
+  }
```

Comment:
`ret==CE_Warning` is not an error here, just an indication that statistics are not present.
You can change the signature of the function to return an int indicating this, and then use
```
  if (ret != 0 && ret != CE_Warning) {
    forceCPLError(ctx,ret);
  }
  godalUnwrap();
  return (ret==0);
```